wordpress-mobile/WordPress-iOS
# Update WordPressAuth pod for password autofill

Fixes #7615

To test:
- Build and run on device
- Attempt to login using password autofill, from keychain storage
- Ensure it succeeds
- The same, but this time with an incorrect password stored
- Ensure it fails and presents the option to enter a different password

Update release notes:
